index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+</head>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<link rel="stylesheet" href="styles.css">
+<body>
+    <header class="header">
+    <nav class="navbar">
+        <a href="#" class="nabar-brand"> My logo</a>
```

Comment:
- Hey, please, kindly move the logo out of the nav. it is fine for the logo to be inside the header. let's improve on this to improve our page layout.